### PR TITLE
Issue #8657: Modification of javadoc example markers to support metadata generation - modifier

### PR DESCRIPTION
**Pull request by gaurabdg**
#8657

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/modifier/InterfaceMemberImpliedModifierCheck.java [hunk 1]
```diff
@@ -137,6 +137,9 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/modifier/InterfaceMemberImpliedModifierCheck.java:141]: Combine with next paragraph

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/modifier/ClassMemberImpliedModifierCheck.java [hunk 2]
```diff
@@ -67,6 +67,9 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
+ * <p>
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/modifier/ClassMemberImpliedModifierCheck.java:72]: Combine with next paragraph

Comment (gaurabdg) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/modifier/ClassMemberImpliedModifierCheck.java]: Done

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/modifier/InterfaceMemberImpliedModifierCheck.java (hunk 1 above)
Comment (gaurabdg) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/modifier/InterfaceMemberImpliedModifierCheck.java]: Done

**Comment by romani** [comment]
@gaurabdg ,
please fix CI failure https://travis-ci.org/github/checkstyle/checkstyle/jobs/717240997#L803

**Comment by gaurabdg** [comment]
> @gaurabdg ,
> please fix CI failure https://travis-ci.org/github/checkstyle/checkstyle/jobs/717240997#L803

@romani Done

**Pull request merged**